mocca-client/src/test/java/com/paypal/mocca/client/MoccaClientQueryTest.java

```diff
@@ -223,7 +225,7 @@ public void queryOptionalTest() {
         assertTrue(superComplexResponse.getOptionalField().isPresent());
         assertEquals(superComplexResponse.getOptionalField().get(), "love");
         assertEquals(superComplexResponse.getDuration(), Duration.ofHours(3));
-        assertEquals(superComplexResponse.getUuid(), UUID.fromString("229c07ba-04bc-49a6-13bc-165e1a54cb33"));
+        assertEquals(superComplexResponse.getUuid(), UUID.fromString("229c07ba-04bc-49a6-13bc-165e1a54cb32"));
```

Comment:
Out of curiosity, why did this change?

Reply:
As am debugging the stub mismatch I was trying few things. One is this to differentiate it from other complex test. Just for my help. I will roll it back to previous state once I figure out the stub issue.

Reply:
Reverted the value.